packages/rocketchat-file-upload/server/methods/sendFileMessage.js

```diff
@@ -32,42 +32,46 @@ Meteor.methods({
 			title_link_download: true
 		};
 
-		if (/^image\/.+/.test(file.type)) {
-			attachment.image_url = fileUrl;
-			attachment.image_type = file.type;
-			attachment.image_size = file.size;
-			if (file.identify && file.identify.size) {
-				attachment.image_dimensions = file.identify.size;
+		Meteor.wrapAsync(FileUpload.resizeImagePreview(file, Meteor.bindEnvironment(function(base64Preview) {
```

Comment:
You could check if file is image and call this only for images, makes more sense to me.